<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToolDialog</class>
 <widget class="QDialog" name="AddToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>242</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add File...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileName">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>143</height>
          </size>
         </property>
         <property name="title">
          <string>Project</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QRadioButton" name="inCornucopia">
              <property name="minimumSize">
               <size>
                <width>115</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Cornucopia</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inTools">
              <property name="minimumSize">
               <size>
                <width>115</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Tools</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inDemoUI">
              <property name="minimumSize">
               <size>
                <width>115</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>DemoUI</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inTest">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="header">
           <property name="text">
            <string>.h</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="source">
           <property name="text">
            <string>.cpp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="includeMoc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Generate inclusion of moc file in the cpp file</string>
           </property>
           <property name="text">
            <string>moc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>inCornucopia</tabstop>
  <tabstop>inTools</tabstop>
  <tabstop>inDemoUI</tabstop>
  <tabstop>header</tabstop>
  <tabstop>source</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inTest</sender>
   <signal>toggled(bool)</signal>
   <receiver>header</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inTest</sender>
   <signal>toggled(bool)</signal>
   <receiver>includeMoc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inCornucopia</sender>
   <signal>toggled(bool)</signal>
   <receiver>includeMoc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
